examples/community/stable_diffusion_tensorrt_txt2img.py

```diff
@@ -918,6 +925,7 @@ def __call__(
 
             # Pre-initialize latents
             num_channels_latents = self.unet.in_channels
+            # Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline
```

Comment:
Good find! But violation of the 5th and 6th rules.

Reply:
Soo, I need to copy it's implementation/whole code from `StableDiffusionPipeline` and paste it into `TensorRTStableDiffusionPipeline` and add that comment right?

Reply:
Exactly!

Reply:
Exactly! See [here](https://github.com/huggingface/diffusers/pull/7071/files#r1508008109).

Reply:
Have I done it right?

Reply:
Yes! Something is missing in the "# Copied from ..." parts, find them :)

Reply:
Do I need to mention the function name as well? Like "# Copied from file.function_name" ?